src/main/kotlin/no/nav/familie/ef/sak/validering/FagsakPersontilgang.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.familie.ef.sak.validering
+
+import no.nav.familie.ef.sak.api.fagsak.FagsakRequest
+import no.nav.familie.ef.sak.integration.FamilieIntegrasjonerClient
+import no.nav.familie.ef.sak.service.PersonService
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import javax.validation.ConstraintValidator
+import javax.validation.ConstraintValidatorContext
+
+@Component
+class FagsakPersontilgang(private val integrasjonerClient: FamilieIntegrasjonerClient,
+                          private val personService: PersonService)
+    : ConstraintValidator<FagsakPersontilgangConstraint, FagsakRequest> {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    override fun isValid(fagsakRequest: FagsakRequest, ctx: ConstraintValidatorContext): Boolean {
+        val identifikatorer = personService.hentPersonMedRelasjoner(fagsakRequest.personIdent).identifikatorer()
+        integrasjonerClient.sjekkTilgangTilPersoner(identifikatorer)
+                .filterNot { it.harTilgang }
+                .forEach {
+                    logger.error("Bruker har ikke tilgang: ${it.begrunnelse}")
```

Comment:
Hva er det som typisk står i begrunnelsen her? Er vi sikre på at det aldri kan være noen personopplysninger der? Hvis ikke bør vi logge til secure logger 🙂

Reply:
hmm. Det har jeg ikke tenkt over, men kan godt legge begrunnelsen i secure-logger, ja!

Reply:
Vi kan nok aldri være sikkre på det, idag fylles den kun i fra AdRolle i integrasjoner, som kommer fra config "no.nav.security.jwt.tilgang", kanskje det er tilsrekkelig å kun logge dette uten it.begrunnelse?